package.json

```diff
@@ -83,9 +83,9 @@
     {
       "name": "All modules",
       "import": {
-        "./index.js": "{ startKeyUX, hotkeyKeyUX, pressKeyUX, focusGroupKeyUX, jumpKeyUX, hiddenKeyUX, likelyWithKeyboard, getHotKeyHint, hotkeyOverrides }"
+        "./index.js": "{ startKeyUX, hotkeyKeyUX, pressKeyUX, focusGroupKeyUX, jumpKeyUX, hiddenKeyUX, likelyWithKeyboard, getHotKeyHint, hotkeyOverrides, hotkeyMacCompat }"
       },
-      "limit": "1996 B"
+      "limit": "2056 B"
```

Comment:
Let’s update docs and mention `2K`

Reply:
I think it already says that: https://github.com/ai/keyux/blob/96e9bcc9ad17cc88ccd35634a7db08de4d6457e2/README.md?plain=1#L14